### ISSUE TITLE
bug: Two broken links on ananke page

### ISSUE THREAD

**Issue by chris-solacehost** [theNewDynamic/gohugo-theme-ananke#810]
## Steps to Reproduce: Visit https://themes.gohugo.io/themes/gohugo-theme-ananke/

1. https://themes.gohugo.io/themes/gohugo-theme-ananke/
2. https://themes.gohugo.io/tags/dark%20mode/
3. https://formspree.io/your@email.com&#34 -> https://formspree.io/your@email.com&

## The bigger picture (we need context)

<details><summary>Hugo Config</summary>
<p>

</p>
</details>

**Comment by davidsneighbour** [comment]
Yep, that needs some setup on the demo page side, but also a fallback for not properly set up systems.

**Comment by chris-solacehost** [comment]
Happy to report.

**Comment by davidsneighbour** [comment]
I am closing this ;)

I though we were talking about the demo page of the theme, not the page in the theme collection of gohugo.io. 

- dark mode - seems to have been an issue with the gohugo site, not Ananke
- formspree link - obviously not working in a code sample. You have to set that up for your own email address/form link.
